assets/js/translator.js

```diff
@@ -449,6 +562,44 @@ function translateText() {
                     if(data.responseStatus === 200) {
                         $('#translatedText').html(data.responseData.translatedText);
                         $('#translatedText').removeClass('notAvailable text-danger');
+
+                        $('#translatedText').attr('pristineText', data.responseData.translatedText);
+
+                        if(config.SUGGESTIONS.enabled) {
+                            var localizedTitle = dynamicLocalizations['Suggest_Title'];
+                            var placeholder = dynamicLocalizations['Suggest_Placeholder'];
+                            $('#suggestedWordInput').attr('placeholder', placeholder);
+                            $('#translatedText').html(                                
+                                $('#translatedText').html().replace(/(^|\W|\d)(\*|\@|\#)(\w+)/g, '$1<span class="wordSuggestPop text-danger" title="' + localizedTitle + '">$3</span>'));
+                        }
+
+                        $('#translatedTextClone').html(
+                            $('#translatedText').attr('pristineText'));
+
+                        $('.wordSuggestPop').click(function() {
```

Comment:
`wordSuggestPop` => `wordSuggestPopover` and `wordSuggestPopInline` => `wordSuggestPopoverInline`